### DISCUSSION TITLE
Created by and Updated by is not showing in the tables

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#5718] [Help]
I have encountered with an incident created by and updated by are not showing 
<img width="807" alt="created" src="https://user-images.githubusercontent.com/94033860/166812917-5bc8321f-2f9a-4272-8995-1eb0bbb0b7a4.png">

**Comment by melohagan** [comment]
Were the rows created through automation, or was the table imported?

**Reply by ghost** [reply]
@melohagan The rows are created/ updated by using button actions >> Save row . I tried setting the columns created by / updated by with current user.email and this didn't work

**Reply by melohagan** [reply]
The little wand icon indicates that the field is auto-generated, so you shouldn't really ever be manually setting these. 

Is a user definitely logged in when these buttons are being pressed; it's not an anonymous access? Also, is that user definitely in the Users table?

**Reply by ghost** [reply]
@melohagan I checked the user is logged in and using the action button via a published app url . User is present in user table

**Reply by xnohat** [reply]
These Auto Columns are reserved by platform, it's mean that you should not update by yourself, platform will handle it

**Reply by ghost** [reply]
@xnohat Got your point, since the update by column is not getting filled , i tried to update that with a button action but that also didn't work .

**Reply by melohagan** [reply]
@newtonlaw95 If you have a look at the network tab when you click the save button, do you see any errors?

**Reply by ghost** [reply]
@melohagan there were no errors

**Reply by melohagan** [reply]
Would you be able to share your app export?

**Comment by Bytelegion** [comment]
Created by User not populated when using automations to create row ?

Anyone facing same issue ?

**Reply by Bytelegion** [reply]
If using API , the right user populated , but in case of automations [webhooks], created user column not populated

**Reply by melohagan** [reply]
Hi @Bytelegion 

This is intended behaviour, as the create row is done by the automation and not a user.

**Reply by Bytelegion** [reply]
Any way to populate some user info [e.g a bot account]

**Reply by melohagan** [reply]
Not currently at the moment. The field is an autogenerated one, so you can't set the value. It might be worth raising a feature request if you think it'd be useful!

**Reply by Bytelegion** [reply]
Thanks will do that
